vp/devel/python_peripherals/DLA.py

```diff
@@ -895,7 +907,7 @@ def get_input_data(self):
                     column_wise.append(data[idx])
 
         column_wise = reshape(column_wise, (channels, height, width))
-        print_matrix(column_wise[0], "flat input:", pformat="decimal")
+        #print_matrix(column_wise[0], "flat input:", pformat="decimal")
```

Comment:
Only-slight-frown upon the commented out print. I do realize there's not really a good way to configure whether a print shall be on or not with this infra, apart from commenting out.

Reply:
I don't think these kinds of changes should be introduced as a part of a new feature anyways. This one accidentally slipped past. Change removed.